tests/api/fixtures/text.js

```diff
@@ -56,8 +56,19 @@ const randomWords = (numberOfWords, suffix = '') => _.sampleSize(lorem, numberOf
 
 const firstName = () => _.sampleSize(firstNames, 1)[0]
 
+const randomWord = () => randomWords(1)
+
+const distinctWords = () => {
```

Comment:
i would replace `distinctWords` by `randomWords` to only have `randomWords` which are distinct

Reply:
done in 3ca978e2